# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/chromium/src/chrome/common/ipc_channel_posix.cc
# Commit: 110b2384e7d1
# Full Hash: 110b2384e7d11cd70daa2ea0d06f51926642bdc2
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2021-06-22 21:29:07
# Regressor Bug: 1706374
# File Overlap Count: 1
# Description:
#   Bug 1706374 - Part 9: Allow reading OtherPid() from an opened IPC::Channel, r=handyman
#   
#   This will be used to handle pre-opened channels more reliably in ports code.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112774
# ==============================================================================

diff -r ab2b086abbd4 -r 110b2384e7d1 ipc/chromium/src/chrome/common/ipc_channel_posix.cc
--- a/ipc/chromium/src/chrome/common/ipc_channel_posix.cc	Mon Jun 21 21:53:09 2021 +0000
+++ b/ipc/chromium/src/chrome/common/ipc_channel_posix.cc	Mon Jun 21 21:53:09 2021 +0000
@@ -965,6 +965,8 @@
   channel_impl_->CloseClientFileDescriptor();
 }
 
+int32_t Channel::OtherPid() const { return channel_impl_->OtherPid(); }
+
 bool Channel::Unsound_IsClosed() const {
   return channel_impl_->Unsound_IsClosed();
 }